fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/api/FHIRDbDAO.java

```diff
@@ -21,32 +21,18 @@
     public static final String PROPERTY_DB2_PSWD = "password";
 
     /**
-     * Acquires and returns a JDBC database connection to the FHIR database.
-     * If no DB Properties are available, an attempt is made to acquire the connection via a Datasource obtained via
-     * JNDI. If DB Properties are present, those properties are used to build the Connection.
+     * Obtains a database connection. Connection is configured and ready to use. Its
+     * schema will be set to the configured FHIR data schema (usually 'FHIRDATA') and
+     * if multi-tenant, the tenant property will have been set.
      * 
      * @return Connection - A connection to the FHIR database.
      * @throws FHIRPersistenceDBConnectException
      */
     Connection getConnection() throws FHIRPersistenceDBConnectException;
 
-    /**
-     * Returns a previously set externally managed DB connection, used by the DAO for all DB activity.
-     * 
-     * @return Connection
-     */
-    Connection getExternalConnection();
-
-    /**
-     * Sets an externally managed DB connection, used by the DAO for all DB activity.
-     * 
-     * @param connection
-     */
-    void setExternalConnection(Connection connection);
-
     /**
      * @return true if this DAO is connected to a DB2 database.
      * @throws Exception
      */
-    boolean isDb2Database() throws Exception;
+    boolean isDb2Database();
```

Comment:
is this necessary to have here? it reads as if this can be refactored out, which would make this interface cleaner

Reply:
Do you mean the isDb2Database? We can look at removing as part of the FhirDbDAO property cleanup.

Reply:
It's used by some of the query constructor classes to tweak syntax. This should be handled by the translator, but it's inline for now - and not something that's part of issue-1144.

Reply:
Right, it didn't seem necessary after the code changes.